platform/feature-api/communication/postoffice/src/protocol.ts

```diff
@@ -2,19 +2,19 @@
  * This module defines the protocol used for translating function calls into bare objects. Users rarely need to use this
  * module directly.
  *
- * The [[endpointClient]] and [[endpointServer]] functions speak a simple protocol:
+ * The [[backendClient]] and [[backendServer]] functions speak a simple protocol:
  *
- * - each call is represented as an [[EndpointRequestPart]]
- * - a chain of calls is represented as an [[EndpointRequest]]
- * - a response is any kind of value ([[EndpointResponse]])
+ * - each call is represented as an [[BackendRequestPart]]
+ * - a chain of calls is represented as an [[BackendRequest]]
+ * - a response is any kind of value ([[BackendResponse]])
  *
  * @packageDocumentation
  */
 
 /**
  * A function call representing the name of a method (of type string) and an array of arguments (of any type).
  */
-export interface EndpointRequestPart {
+export interface BackendRequestPart {
```

Comment:
Alternative: "RequestAtom" or "BackendRequestAtom" or "SingleRequest"

Reply:
I don't really mind this naming, an array of parts is indeed a whole :D

Reply:
I believe they're calling it "part" kinda referring to "multipart", but I'm not sure. Would need some closer studying if we want to change terminology, like I said above.